gce/image/scylla_gce.json

```diff
@@ -11,7 +11,7 @@
       "machine_type": "{{user `instance_type`}}",
       "metadata": {"block-project-ssh-keys": "TRUE"},
       "image_family": "scylla",
-      "image_name": "scylla-{{user `scylla_branch_version`| clean_resource_name}}-build-{{user `scylla_build_id`| clean_resource_name}}",
+      "image_name": "scylla-{{user `scylla_image_name`}}-build-{{user `scylla_build_id`}}",
```

Comment:
why did you remove `clean_resource_name` it 

> clean_resource_name will convert upper cases to lower cases and replace illegal characters with a "-" character. 

https://www.packer.io/docs/templates/legacy_json_templates/engine#clean_resource_name

Reply:
By mistake. fixed